metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatEventBus.java

```diff
@@ -38,24 +35,20 @@
 @Slf4j
 public class MetacatEventBus {
 
-    private final ApplicationEventPublisher eventPublisher;
-    private final ApplicationEventMulticaster eventMulticaster;
+    private final MetacatApplicationEventMulticaster applicationEventMulticaster;
     private final Counter eventAsyncCounter;
     private final Counter eventSyncCounter;
     /**
      * Constructor.
      *
-     * @param eventPublisher   The synchronous event publisher to use
-     * @param eventMulticaster The asynchronous event multicaster to use
+     * @param applicationEventMulticaster The event multicaster to use
      * @param registry         The registry to spectator
      */
     public MetacatEventBus(
-            @Nonnull @NonNull final ApplicationEventPublisher eventPublisher,
-            @Nonnull @NonNull final ApplicationEventMulticaster eventMulticaster,
-            @Nonnull @NonNull final Registry registry
+        @Nonnull final MetacatApplicationEventMulticaster applicationEventMulticaster,
```

Comment:
Didn't we want to put these `@Nonnull` at the package level and leave in the Lombok ones cause they're runtime catches?

Reply:
Yeah I will revert this change. If I have to do it at the package level, a lot of files need to change. Will do it in another PR.